sample-code/package.json

```diff
@@ -0,0 +1,21 @@
+{
+	"type": "module",
+	"name": "sample-code",
+	"version": "0.0.1",
+	"dependencies": {
+        "@sap-ai-sdk/gen-ai-hub": "workspace:*",
+		"@types/express": "^4.17.21",
```

Comment:
The formatting seems a bit off

Reply:
Can we also lint this project?

Reply:
I think we can, I added a config now. But I don't know how to fix this linting errors:

```
/ai-sdk-js/sample-code/src/aiservice.ts
  1:30  error  Parse errors in imported module '@sap-ai-sdk/gen-ai-hub': parserPath or languageOptions.parser is required! (undefined:undefined)  import/namespace

/ai-sdk-js/sample-code/src/index.ts
  15:5  error  Unexpected console statement  no-console
  30:5  error  Unexpected console statement  no-console
  42:3  error  Unexpected console statement  no-console
  ```
  
I guess the no-console thing is a rule to enforce proper logging? For the first error, I have no idea..